### PR TITLE
Converted table to list

### PR DESCRIPTION
**Pull request by tupes**
The items and cart items aren't really in a tabular format, so I converted them to lists.

### CODE REVIEW

**Comment by JoshMacSween** [comment]
This code sucks. Fix it.

File: src/CartTable.js [hunk 1]
```diff
@@ -17,14 +17,10 @@ export default function CartTable() {
   ];
 
   return (
-    <table>
-      <tbody>
-        {items.map((item) => (
```
Comment (JoshMacSween) [inline src/CartTable.js:22]: What are you doing here? What is a map.

**Review by JoshMacSween — APPROVED** [review]
Great job

File: src/CartTable.js [hunk 2]
```diff
@@ -17,14 +17,10 @@ export default function CartTable() {
...
-          <tr key={item.id}>
-            <td>{item.name}</td>
-          </tr>
-        ))}
-      </tbody>
-    </table>
+    <ul>
```
Comment (mirehabte) [inline src/CartTable.js:20]: I would say it is better if you can use a div tag.

**Pull request merged**